roundtripper.go

```diff
@@ -141,7 +141,7 @@ func (rt *roundTripper) dialTLS(ctx context.Context, network, addr string) (net.
 		host = rt.serverNameOverwrite
 	}
 
-	tlsConfig := &tls.Config{ClientSessionCache: rt.clientSessionCache, ServerName: host, InsecureSkipVerify: rt.insecureSkipVerify, OmitEmptyPsk: true}
```

Comment:
rt.transportOptions can be nil if not provided explicit via client options.

Reply:
Good catch, thanks.